manta-trusted-setup/src/groth16/ceremony/config/ppot.rs

```diff
@@ -371,6 +416,201 @@ where
     .await
 }
 
+/// Configuration for the Groth16 Phase2 Server.
+#[derive(Clone, Default)]
+pub struct Config(Ed25519<RawMessage<u64>>);
+
+impl Pairing for Config {
+    type Scalar = bn254::Fr;
+    type G1 = bn254::G1Affine;
+    type G1Prepared = <bn254::Bn254 as PairingEngine>::G1Prepared;
+    type G2 = bn254::G2Affine;
+    type G2Prepared = <bn254::Bn254 as PairingEngine>::G2Prepared;
+    type Pairing = bn254::Bn254;
+
+    #[inline]
+    fn g1_prime_subgroup_generator() -> Self::G1 {
+        bn254::G1Affine::prime_subgroup_generator()
+    }
+
+    #[inline]
+    fn g2_prime_subgroup_generator() -> Self::G2 {
+        bn254::G2Affine::prime_subgroup_generator()
+    }
+}
+
+impl Size for Config {
+    const G1_POWERS: usize = (Self::G2_POWERS << 1) - 1;
+    const G2_POWERS: usize = 1 << 17;
+}
+
+impl kzg::Configuration for Config {
```

Comment:
I checked and it compiles fine without the `impl kzg::Configuration` here.  Which makes sense since this file only needs to configure phase 2.

Reply:
And I think this file should be renamed to something like `.../config/bn254.rs` because actually once you remove `impl kzg::Configuration` there's nothing left here that refers specifically to the PPoT ceremony (but please double check that).  The only concrete choices we make in this file are Ed25519 as signature scheme and Bn254 as pairing curve.

Reply:
The first two: agree, I deleted the impl already. The third: I also agree, but I'd rather do it on the next PR which has been forked from this one.